src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionDetails.cs

```diff
@@ -592,31 +592,41 @@ public string AzureAccountToken
         }
 
         public bool IsComparableTo(ConnectionDetails other)
```

Comment:
I'd suggest adding a doc comment to this explaining what properties it is/isn't comparing and what the general purpose of this is so others know what to expect.